src/markdown-to-draft.js

```diff
@@ -147,7 +156,7 @@ function parseInline(inlineItem, BlockEntities, BlockStyles) {
 
       blockEntityRanges.push({
         offset: strlen(content) || 0,
-        length: 0,
+        length: 1,
```

Comment:
Hello! Sorry for the delay in review and unfortunately I can't guarantee that I'll reply to your response in a timely manner, things have been very very busy for me with the baby, much busier than I expected!

Anyway...

I am wondering about the change here and the change at https://github.com/Rosey/markdown-draft-js/pull/49/files#diff-6ed54c3ce52b221b0eb0f7b22578bfe4R232

If I revert this back to `0` and the other line back to `''` the tests still pass (provided I update the tests as well), so I was curious if you could provide an explanation for why we need the change to length/content? I haven't used images at all in draftjs so I'm not familiar with the details 😄